tests/phpt/expr/instanceof.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+test variables and operators expressions
+--FILE--
+<?php
+
+macro {
+   ·expr()·my_expr
+} >> {
+expression {
+    ·my_expr
+}
+}
+
+$a instanceof \Some\Large\FQCN
+
+?>
+--EXPECTF--
+<?php
+
+expression {
+    $ainstanceof\Some\Large\FQCN
```

Comment:
It seems that whitespace should be preserved in the Ast every time we get an operator that is also a valid identifier, so expansion won't break like in this case `$ainstanceof\Some\Large\FQCN`. Example.:

```php
$instanceof = chain(
    variable()->as('left'),
    indentation(), # preserves the whitespace :)
    token(T_INSTANCEOF)->as('operator'),
    indentation(), # preserves the whitespace :)
    ns()->as('right')
)
->as('expr');

$ts = \Yay\TokenStream::fromSource('<?php $a  instanceof     \Some\Large\FQCN');

\Yay\token(T_OPEN_TAG)->parse($ts); // get rid of <?php

$ast = $instanceof->parse($ts);

$src = '';
$nodes = $ast->array();
array_walk_recursive($nodes, function($n) use(&$src){ $src .= $n; });
var_dump($src); #>>  "$a  instanceof     \Some\Large\FQCN"
```

Reply:
fixed in f60fe4f